src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/Example1.txt

```diff
@@ -13,6 +13,11 @@ public class Test {
      */
     private void methodWithValidCommentStyle() {}
 
+    /**
+     * {@return something} // OK, @return tag automatically inserts a period after the text
+     */
+    private void methodWithValidCommentStyle2() {}
```

Comment:
This example doesn't make sense; we would never have a return tag on a void method. Please update this method to return a `String`

Reply:
and make it somthing like:
```
// ok below, @return tag automatically inserts a period after the text
/**
     * {@return something} 
     */
```

to keep javadoc comment "realistic" and all hints text outside to never affect validatin.

Reply:
@0xbakry please also update the first item under “ The following checks are performed:” in the check description

Reply:
Totally agree!! I have missed this, sorry. 
Done.

Reply:
> @0xbakry please also update the first item under “ The following checks are performed:” in the check description

Done.